railties/lib/rails/generators/rails/scaffold/scaffold_generator.rb

```diff
@@ -8,7 +8,7 @@ class ScaffoldGenerator < ResourceGenerator # :nodoc:
       remove_hook_for :resource_controller
       remove_class_option :actions
 
-      class_option :api, type: :boolean
+      class_option :api, type: :boolean, desc: "Generate API controller"
```

Comment:
I see this matches the [description on `scaffold_controller`](https://github.com/rails/rails/blob/9ff7c9c0231b147486dfdd38325932c95604f1ce/railties/lib/rails/generators/rails/scaffold_controller/scaffold_controller_generator.rb#L15-L16), but would it be helpful to elaborate more?

```suggestion
      class_option :api, type: :boolean,
        desc: "Generate API-only controller and tests, with no view templates"
```

Reply:
Thank you for your improvement!